lua/telekasten.lua

```diff
@@ -2181,6 +2181,8 @@ local function FollowLink(opts)
             -- we are in a link
             vim.cmd("normal yi]")
             title = vim.fn.getreg('"0')
+            -- this should remove link alias
+            title = string.sub(title, 0, string.find(title, "|") - 1)
```

Comment:
Probably better to use the pattern `%s*|`, so we can remove spaces that would be before the vertical pipe in case the link is formatted like `[filename | alias]`.

Reply:
Good point! Thanks a lot!